index.js

```diff
@@ -30,6 +30,26 @@ module.exports = function searchWithYourKeyboard (inputSelector, hitsSelector) {
     const hits = getCurrentHits()
     const queryExists = Boolean(input && input.value && input.value.length > 0)
 
+    if (event.ctrlKey && event.keyCode === 13 && activeIndex > 0) {
```

Comment:
Instead of having this standalone `if` statement up here, why not just check for `event.ctrlKey` in the `case 'enter'` below. Something like this:

```js
if (event.ctrlKey) {
  window.open(href, '_blank')
} else {
  window.location = href
}